signer/fourbyte/4byte.go

```diff
@@ -127472,7 +127472,7 @@ var __4byteJson = []byte(`{
 "de7bea74": "getThreshold(uint8)",
 "de7c2ed2": "TEAM_VESTING_DURATION()",
 "de7c99c3": "releasePrivateBonusTokens(address[],uint256[])",
-"de7cb327": "Proof()",
+"de7cb327": "StorageProof()",
```

Comment:
how to know whether we should change it? which part it's related to?

Reply:
That's a mistake, will be changed.